src/stories/SearchInput.story.js

```diff
@@ -24,6 +24,7 @@ stories
         <StoryItem>
           <SearchInput
             expandable={boolean('Expandable', false)}
+            defaultExpanded={boolean('Default expanded', false)}
```

Comment:
Maybe I misunderstand this prop, but wouldn't simply the `expanded` name be enough? It's not only default, could be manipulated during its lifecycle right?

Reply:
Probably forceExpanded will suit best